src/Http/Request/Body/JsonSchemaValidatingRequestBodyParser.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Request\Body;
+
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\SchemaError;
+use Opis\JsonSchema\Errors\ErrorFormatter;
+use Opis\JsonSchema\Validator;
+use Psr\Http\Message\ServerRequestInterface;
+
+final class JsonSchemaValidatingRequestBodyParser implements RequestBodyParser
+{
+    private const SCHEMA_LOCATIONS = __DIR__ . '/../../../../vendor/publiq/stoplight-docs-uitdatabank/models/';
```

Comment:
Maybe we can add one PHP file inside `publiq/stoplight-docs-uitdatabank` with some constants for file names and file locations? Then we have a bit more flexibility inside the `publiq/stoplight-docs-uitdatabank` package on where to store something without having to modify these hardcoded paths.

Reply:
As discussed I added a file with constants in this PR instead.